pageflow-before-after.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'pageflow', ['>= 0.11', '< 13']
-  spec.add_dependency 'pageflow-public-i18n', '~> 1.0'
+  spec.add_dependency 'pageflow-public-i18n', '>= 1.8'
```

Comment:
I am going with pessimistic version constraints for all major version gems now. I think this is the safe thing to do when following SemVer. All other Pageflow gems also depend on `~> 1.x` versions of `pageflow-public-i18n`. At the moment I do not really see there ever being a `2.0` of `pagefow-public-i18n`. But if there ever should be, updating the version requirement seems fine to me as a way of explicitly stating that compatibility has been checked.